services/data/src/utils/queryToResourceUrl.ts

```diff
@@ -3,6 +3,8 @@ import {
     QueryParameters,
     QueryParameterValue,
 } from '../types/Query'
+import { ContextInput, ContextType } from '../types/Context'
+import path from 'path'
```

Comment:
This is run by the browser? Does TypeScript bundle NodeJS built-ins automatically?

I'm used to having to rely on other ways of doing this so it caught my eye.

Reply:
```
(!) node-resolve plugin: preferring built-in module 'path' over local alternative at 'path', pass 'preferBuiltins: false' to disable this behavior or 'preferBuiltins: true' to disable this warning
(!) Unresolved dependencies
https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency
path (imported by ../services/data/build/es/utils/queryToResourceUrl.js)
```

Reply:
Shouldn't `joinPath` work here?